data/pools/transactionPool.go

```diff
@@ -337,6 +342,20 @@ func (pool *TransactionPool) Verified(txn transactions.SignedTxn) bool {
 	return pendingSigTxn.Sig == txn.Sig && pendingSigTxn.Msig.Equal(txn.Msig) && pendingSigTxn.Lsig.Equal(&txn.Lsig)
 }
 
+// EvalOk for LogicSig Eval of a txn by txid, returns the SignedTxn, error string, and found.
+func (pool *TransactionPool) EvalOk(txid transactions.Txid) (tx transactions.SignedTxn, txErr string, found bool) {
```

Comment:
nit: The first return value is not being used, and can be omitted from the function signature; also, the second argument need to return just whether it's != "" or not. ( i.e. error'ed or not ).

Reply:
dropped the unused argument, but there's one place we return the error string if it's not ""